test/util/e2e-helpers.js

```diff
@@ -37,8 +37,8 @@ function overrideVetsGovApi(client) {
 }
 
 module.exports = {
-  baseUrl: 'http://localhost:3333',
-  apiUrl: 'http://localhost:4000',
+  baseUrl: `http://localhost:${process.env.WEB_PORT || 3333}`,
+  apiUrl: `http://localhost:${process.env.API_PORT || 4000}`,
```

Comment:
@akainic: How is that getting initialized for these `vets-website` tests? Or is it not getting initialized atm? Is that something that needs to get added to `master` here?